cmd/larry/main.go

```diff
@@ -49,24 +59,25 @@ func main() {
 			{Name: "@kannav02"},
 			{Name: "@siddhant-k-code", Email: "siddhantkhare2694@gmail.com"},
 			{Name: "@savagedev"},
+			{Name: "@till"},
 		},
 		Action: func(c *cli.Context) error {
 			prov, err := getProvider(cfg)
 			if err != nil {
-				log.Fatal(err)
+				return err
```

Comment:
Returning these errors as you have a fatal after `app.Run()`

Reply:
Yes, that will work :)